romancal/source_catalog/tests/test_source_catalog.py

```diff
@@ -381,7 +381,7 @@ def test_do_psf_photometry(tmp_path, image_model):
     # check the number of sources that have been detected
     assert len(cat) == 7
     # check that all sources have both position and flux determined (ignore errors/flags)
-    assert all(len(cat[x]) and cat[x] is not [None, np.nan] for x in psf_colnames)
+    assert all(len(cat[x]) and cat[x] != [None, np.nan] for x in psf_colnames)
```

Comment:
I suspect we want "not in" rather than !=, right, or do I misunderstand?  Actually, cat is a Table, so cat[x] should be a numpy array, so we probably want something like np.all(np.isfinite(cat[x]))?